launch/test_planner.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+    <arg name="platform" default="sim" doc="string representing the platform" />
+    <node name="test_planner_abstract" pkg="iarc7_abstract" type="test_planner.py" />
+    <param name="safety_bonds/test_planner_abstract/form_bond" value="true" />
+
+    <include file="$(find iarc7_launch)/launch/main.launch">
```

Comment:
I don't think we want `main` here, we probably want `passive_stack` with some other stuff

Reply:
Main launches `passive stack` and the motion stack.

Reply:
But it won't in the future, that's just what it does now because that's what we have implemented.

Reply:
In the future it'll launch a bunch of AI stuff and possibly other things you don't need.

Reply:
fixed